docs/edgeql/ddl/migrations.rst

```diff
@@ -22,26 +22,14 @@ Create a new migration.
         [ ... ]
     "}" ;
 
-    [ WITH [ <module-alias> := ] MODULE <module-name> ]
-    CREATE MIGRATION <name> "{"
-        <ddl-command> ;
-        [ ... ]
-    "}" ;
-
 
 Description
 -----------
 
-``CREATE MIGRATION`` defines a new schema migration for a specific module.
-If *name* is qualified with a module name, then the migration is created
-for that module, owtherwise it is created for the current module, as
-determined by the session or the ``WITH`` block.
-
-There are two forms of ``CREATE MIGRATION`` as shown in the synopsis above.
-The first form uses a specific description of the target schema state and
-generates the necessary DDL commands automatically based on the current and
-the target state.  The second form uses explicit DDL command specifications
-for the migration.
+``CREATE MIGRATION`` defines a migration of the schema to a new state.
+The target schema state is described using :ref:`SDL <ref_eql_sdl>`
+and the migration generates the necessary :ref:`DDL <ref_eql_ddl>`
```

Comment:
I know that it's not a new text, but I still don't get it: why we say DDL commands are generated? I see it as a schema is changed. I have never seen DDL when used CREATE MIGRATION.

Reply:
I'd fix this by replacing `and the migration generates` with `and the migration generates ... behind the scenes`

Reply:
I've changed the wording to "behind the scenes".

We used to have a command `GET MIGRATION` that actually allowed to see the generated DDL, but lots of things since then have changed. The command is currently broken, but we might have some equivalent way to view the representation of the migration in the future.

Reply:
> The command is currently broken

Is it? I think we have a few tests that depend on it working.

Reply:
Well it's sufficiently broken that when I tried it in REPL with a simple migration it failed (I think it ultimately failed to find some name in the schema, although I don't recall the error). But the main point is that we don't test it too much even if we test it at all, so it's not surprising that it would be broken in some cases.